src/oauth2.cc

```diff
@@ -139,6 +140,17 @@ std::string Sign(const std::string& data, const PKey& pkey) {
   return std::string(reinterpret_cast<char*>(result.get()), actual_result_size);
 }
 
+} // namespace
+
+OAuth2::OAuth2(const Environment& environment)
+  : OAuth2(environment, ExpirationImpl<std::chrono::system_clock>::New()) {}
+
+OAuth2::OAuth2(const Environment& environment,
+               std::unique_ptr<Expiration> expiration)
+  : environment_(environment),
+    token_expiration_(std::move(expiration)),
+    token_endpoint_(kDefaultTokenEndpoint) {
+  ::google::GceApiRequestErrors();
```

Comment:
What does this do? (Return value is not used.) Thanks.

Reply:
TL;DR: I will add comments on why we need to register the measure here.

This function will register the [measure](https://opencensus.io/stats/measure/#3) if it is not created. And the measure should be registered before the view is registered (it's registered at https://github.com/Stackdriver/metadata-agent/pull/210/files#diff-61b93c57ea92f91ec66fdd4a280d8e8bR38) - otherwise there will be error.

Similar pattern can be found in [grpc codebase](https://github.com/grpc/grpc/blob/7e4be890148cae7d9350007d4372f7492c52569c/src/cpp/ext/filters/census/grpc_plugin.cc#L33-L55) as well.